jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -272,12 +272,22 @@ def step075_customize_puppet_settings(script_dir, puppet_settings) {
     }
 }
 
-def step080_customize_java_args(script_dir, server_java_args, server_era) {
-    if ((server_java_args == null) || (server_java_args == "")) {
+def step080_customize_java_args(script_dir, server_heap_settings, server_era) {
+    if ((server_heap_settings == null || server_heap_settings == "") && "${JAVA_ARGS_ADDITIONS}" == "" && "${HEAP_OVERRIDES}" == "") {
```

Comment:
mimicking the params following, is this the same as `"${server_heap_settings}" == ""`

Reply:
I have no idea. One is a jenkins param and one is a groovy var.

Reply:
oh wait. yea no, not at all. because "${server_heap_settings}" would be "null" if it is null.